packages/quantic/force-app/main/default/lwc/searchBoxStyle/searchBoxStyle.css

```diff
@@ -3,22 +3,46 @@
 .searchbox__container {
   display: flex;
   position: relative;
-  border-color: #e5e8e8;
   min-height: 48px;
   box-sizing: border-box;
   border-radius: 0.25rem;
-  overflow: hidden;
+  overflow: visible;
+  border: var(--lwc-borderWidthThin, 1px) solid var(--lwc-brandAccessible, #0176d3);
+}
+
+.searchbox__container-expandable {
+  display: block;
+}
+.searchbox_floating-container {
+  position: absolute;
+  width: 100%;
+  background-color: var(--lwc-colorBackgroundAlt, rgb(255, 255, 255));
+  border-radius: 0.25rem;
+}
+
+.searchbox__container-wrapper {
+  min-height: 3.125rem;
 }
 
 .searchbox__input {
-  display: flex;
   border-radius: 0.25rem;
   border-bottom-left-radius: 0.25rem;
   position: relative;
   height: 48px;
   box-sizing: border-box;
 }
 
+textarea.searchbox__input {
```

Comment:
This should be in the expandableSearchBoxInput not here
The common style doesn't need to know about the textarea style